spell/spell-reviewer-mainnet-checklist.md

```diff
@@ -205,10 +205,14 @@ Spell Actions (Per Exec Doc):
   * [ ] MKR transfers
     * [ ] Recipient addresses match Exec Doc
     * [ ] Transfer values match Exec Doc
+        * [ ] All values are specified with 2 decimals
```

Comment:
```suggestion
        * [ ] All values are specified with at least 2 decimals
```
As some transfers might need to be more precise than 2 decimals

Reply:
I don't think I ever came across payments with more than 2 decimals. If we don't specify an exact amount, we might end up with the same issue.

Reply:
> I don't think I ever came across payments with more than 2 decimals

2023-03-08 spell is the example from this year: https://github.com/makerdao/spells-mainnet/blob/0a5b428a84de979fc780b5f3f07671e1b0233db4/archive/2023-03-08-DssSpell/DssSpell.sol#L110

> If we don't specify an exact number, we might end up with the same issue

I think the use of `at least` is clear enough to specify that the precision can be higher, but can't be lower. Restricting it on a low side is not something we can decide as it's dependent on the rounding sacrifice in the real world the governance is willing to take. But if they are willing to take it now and fine to strictly setting it to 2, then I'm actually happy to follow it

Reply:
Let's make it less strict, then. We can re-iterate on this item later.